### ISSUE TITLE
TextInput on Android TV does not work with dpad

### ISSUE THREAD

**Issue by cabhara** [react-native-tvos/react-native-tvos#155]
## Description
Using TextInput on Android TV. Can focus on TextInput after wrapping it in TouchableOpacity (otherwise does not focus).
can move through keyboard values with dpad, but cannot select letters, backspace or submit. basically the middle round button on dpad does not do anything. 
Tested on emulator and on google chromecast.
actually dpad does not work for any selection (middle button)?

## React Native version:
react-native-tvos:  0.63.1-4

## Steps To Reproduce
create tv app with TextInput, use dpad for TextInput

## Expected Results
Should be able to focus on TextInput and enter values with dpad.

**Comment by cabhara** [comment]
I logged all fired events with TVEventHandler, and no events fired inside the keyboard, or when pressing the middle dpad button at all.

**Comment by cabhara** [comment]
So actually the select button works on the chromecast device (but not on the emulator), but the keyboard still needs a touchableOpacity to react, and the keyboard will open often even when on other screens. It almost seems the dpad navigates to elements in other react navigation screens, even though the screens are not visible

**Comment by NickAVolpe** [comment]
@cabhara Are you using [enableScreens](https://github.com/software-mansion/react-native-screens#usage-with-react-navigation) with react navigation. That was part of the solution I used to fix navigation issues between screens. (I forget which platform I had done this for)

```
import { enableScreens } from 'react-native-screens';
enableScreens();
```

**Comment by dlowder-salesforce** [comment]
@cabhara it is a known issue that screens that are not visible can still get focus... see #114

**Comment by cabhara** [comment]
@NickAVolpe thanks for the tip! Adding enableScreens() improved my navigation. I still have to add the touchable around the TextInput (I thought that was fixed from reading the docs), but I can use it and navigate around.

**Comment by cabhara** [comment]
> @cabhara it is a known issue that screens that are not visible can still get focus... see #114

@dlowder-salesforce Thanks, I guess I have to go through and make sure that every Button uses the isTVSelectable param.

**Comment by dlowder-salesforce** [comment]
@cabhara please try the latest release (0.63.4-0).... I looked at the fixes done in core since 0.63.1, and it appears that there were some commits around Android issues in TextInput.  See the list of changes: https://github.com/react-native-tvos/react-native-tvos/compare/v0.63.1-4...v0.63.4-0
